pinry-spa/src/assets/locales/en.json

```diff
@@ -0,0 +1,3 @@
+{
+    "loginTitle": "Login"
```

Comment:
Please translate more words if you have time : )

Reply:
not a problem when I have time, I will add more default english translations.

Reply:
@winkidney I have added translated all the placeholders, labels and enclosed tag content within pinry-spa. please can you review the recent changes ?